app/lib/hesa/sex.rb

```diff
@@ -13,7 +13,7 @@ def self.find(sex, cycle_year)
       # The gymnastics here are required because we use 'Prefer not to say' as the value AND label on the SexForm.
       # We have a ticket to change this (the value should be 'information refused'), and when we do, we can simplify this little mess
       # Trello ticket: https://trello.com/c/QN6sBSi9
-      if sex == 'prefer not to say'
+      if sex.downcase == 'prefer not to say'
```

Comment:
We can't use a symbol here because the hesa converter will also use this method and it passes a string here, the one saved on the application_form https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/services/hesa_converter.rb#L12